docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 image or a directory with multiple images. It is designed to be easy to use for both
 non-programmers and programmers.
 
-
-Getting Started
-===============
-
 Installation
-------------
+============
+
 
 Testing
-+++++++
+-------
 
 If you are testing a pre-release version of stellarphot we recommend setting up
 a virtual environment and installing stellarphot in this environment.
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
 
-```bash
-python3 -m venv stellarphot-test
-source stellarphot-test/bin/activate
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `virtualenv`::
+
+
+    python3 -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre --upgrade stellarphot
+
+
+Getting Started
+===============
 
 Overview
 --------
 
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
+   to do this step once if you use the same equipment for all of your observations.
+#. Settings specific to an object need to be made:
+
+   #. night of data, like the photometry aperture radius, need to be made.
+   #. a list of the sources for which you want to perform photometry. These lists can be re-used.
+
+#. Review all of the settings that the photometry routines will use.
```

Comment:
In reviewing the settings a few esthetic notes:
- Can we have "FWHM" and not "Fwhm" in the photometry apertures pane?  This might also have been an issue in the second notebook as well.  I suspect its the result of an algorithm you are using to clean up the strings, but its an acronym, not a word... Maybe your algorithm for standardizing the text appearance could be tweaked to understand "FWHM"?
- Under photometry options, after not really using non-standard abbreviations in the other panes you state "Include Dig Noise" instead of "Include Digitization Noise"? Were you just tight on space for the explanatory text afterward? 
- Another "Fwhm" here instead of "FWHM".

Reply:
> Can we have "FWHM" and not "Fwhm" in the photometry apertures pane? 

fixed

Reply:
> Under photometry options, after not really using non-standard abbreviations in the other panes you state "Include Dig Noise" instead of "Include Digitization Noise"?

The attribute name is `include_dig_noise` and the title is generated from that. I've modified the title to "Include Digitization Noise"